System.Compiler/Nodes.cs

```diff
@@ -7797,7 +7797,24 @@ public override TypeNode DeclaringType
       {
         base.DeclaringType = value;
       }
+    }
+    /// <summary>

```

Comment:
:bulb: this documentation comment is associated with the private field `isTypeArgument`, and the public property `IsTypeArgument` does not have any comment. This can be resolved by moving the backing field before the comment.